flows/finance-data/finance_data_to_s3.py

```diff
@@ -15,23 +18,9 @@
 from datetime import datetime, timedelta
 
 # Prefect
-from prefect import Flow, task, case
-from prefect.storage import GitHub
-from prefect.run_configs import UniversalRun
-from prefect.engine.state import Failed
-from prefect.schedules import Schedule
-from prefect.schedules.clocks import CronClock
-from prefect.backend import set_key_value, get_key_value
-from prefect.triggers import all_successful
-from prefect.tasks.control_flow import merge
-
-from prefect.utilities.notifications import slack_notifier
+from prefect import flow, task, get_run_logger
+from prefect.blocks.system import JSON
 
-# Set up slack fail handler
-handler = slack_notifier(only_states=[Failed])
```

Comment:
can you add the `SlackWebhook` for notifications?

Reply:
There's an Automation in the cloud UI to trigger a slack notification upon flow failure, its configurable to select "all flows" or just certain flows. Charlie set up slack notifications via the UI. 

We could also use the SlackWebhook if we wanted to further refine the error messages or send messages on other states/results.